docs/analytics_tools/local_oracle_db_connections.md

```diff
@@ -0,0 +1,150 @@
+# Local Oracle Database Connections
+
+This document is an aid for setting up a local connection to an Oracle Database using ArcGIS environments, Python Command Prompt, and a local Jupyter Notebook setup.
+
+## First Time Set-up:
+
+**Step 1:** Make sure you have ArcGIS Pro installed on your computer. As part of the ArcGIS Pro installation, Python Command Prompt and Jupyter Notebook should also have been installed as well.
+
+If you do not have ArcGIS Pro installed, submit a SNOW ticket for “ArcPro.”
+
+**Step 2:** Once ArcGIS is installed, open it and go to “Settings” (located at the bottom of the page) and navigate to “Python” on the left blue sidebar.
+
+![How to get to python in ArgGIS](assets/lodc_step2_2.png)
+
+**Step 3:** Go to “Manage Environments” under “Project Environment”
+
+![Locate Manage Environments](assets/lodc_step3.png)
+
+**Step 4:** Clone the environment by clicking the “Clone Default” button next to “Add” -OR- hover beneath the column titled “Clone” until the two squares are shaded.
+
+![Clone Environment](assets/lodc_step4.png)
+
+**Step 5:** Once the cloned environment is complete, select the radio button under “Activate” to activate the cloned environment and click “OK”.
+
+![Select Cloned Environment](assets/lodc_step5.png)
+
+**Step 6:** Then when you are in your new cloned environment, search in the Start Menu for Python Command Prompt and open. Your Python Command Prompt Screen should have the name of your cloned environment (arcgispro-py3-clone) in the command line.
+
+![Open Python Command Prompt](assets/lodc_step6_1.png)
+
+![Make sure you are in cloned environment](assets/lodc_step6_2.png)
+
+**Step 7:** In the Python Command Prompt Command Line enter the following commands to install SQLAlchemy . When prompted if you want to proceed, enter `y` for yes for all commands. This process will take awhile.
+
+```bash
+conda uninstall sqlalchemy
+
+conda install sqlalchemy
```

Comment:
Is there a specific reason to uninstall then reinstall? If you need a more recent version, you could upgrade rather than un/reinstall.